dataland-frontend/tests/e2e/specs/community/SingleDataRequest.ts

```diff
@@ -125,8 +150,11 @@ describeIf(
       cy.get('[data-test="datapoint-framework"]')
         .children()
         .should("have.length", numberOfFrameworks + 1);
-
-      cy.get("[data-test='datapoint-framework']").should("exist");
+      ARRAY_OF_FRAMEWORKS_WITH_VIEW_PAGE.forEach((framework) => {
+        cy.get("[data-test='datapoint-framework']")
+          .should("exist")
+          .should("contain.text", humanizeStringOrNumber(framework));
+      });
```

Comment:
Is it an overhead to get the framework drop-down and check for existence each time? Could it be easily changed to getting and checking it once and afterwards asserting it to contain all the official framework names?

Reply:
no problem, we check only once